### PR TITLE
Python nodes compilation

### PR DESCRIPTION
**Pull request by rasapala**
### 🛠 Summary

JIRA CVS-154716
PYTHON_DISABLE=0 compilation
Moving llm_nodes to separate task.

### 🧪 Checklist

- [ ] Unit tests added.
- [ ] The documentation updated.
- [ ] Change follows security best practices.
``

### CODE REVIEW

File: docs/windows_developer_guide.md [hunk 1]
```diff
@@ -97,7 +117,7 @@ cd model_server
 
 ## COMPILE
 ```
-bazel build --config=windows --jobs=8 --subcommands --repo_env PYTHON_BIN_PATH=C:/opt/Python39/python.exe --verbose_failures --define CLOUD_DISABLE=1 --define MEDIAPIPE_DISABLE=1 --define PYTHON_DISABLE=1 //src:ovms > compilation.log 2>&1
+bazel build --config=windows --jobs=8 --subcommands --repo_env PYTHON_BIN_PATH=C:/opt/Python39/python.exe --verbose_failures --define CLOUD_DISABLE=1 --define MEDIAPIPE_DISABLE=0 --define PYTHON_DISABLE=0 //src:ovms > compilation.log 2>&1
```
Comment (atobiszei) [inline docs/windows_developer_guide.md:120]: Why use windows config simultaneously with defines for MP/PY disable?

Comment (atobiszei) [reply docs/windows_developer_guide.md:120]: Create separate config if needed

File: src/python/pythonnoderesources.hpp [hunk 2]
```diff
@@ -44,7 +48,91 @@ struct PythonNodeResources {
 
     PythonNodeResources(PythonBackend* pythonBackend);
     ~PythonNodeResources();
-    static Status createPythonNodeResources(std::shared_ptr<PythonNodeResources>& nodeResources, const ::mediapipe::CalculatorGraphConfig::Node& graphNode, PythonBackend* pythonBackend, std::string graphPath);
+    static void createOutputTagNameMapping(std::shared_ptr<PythonNodeResources>& nodeResources, const ::mediapipe::CalculatorGraphConfig::Node& graphNodeConfig) {
```
Comment (mzegla) [inline src/python/pythonnoderesources.hpp:51]: Why did you move definition from Cpp to the header?

File: src/python/BUILD [hunk 3]
```diff
@@ -90,7 +90,7 @@ cc_library(
         "pythonexecutorcalculator_cc_proto",
         "utils",
     ],
-    visibility = ["//visibility:private"],
+    visibility = ["//visibility:public"],
```
[the remark this answers is not in the document]
Comment (rasapala) [reply src/python/BUILD:93]: Done when trying to fix some compilation problems with dependencies. It should not be required anymore.

File: src/python/pythonnoderesources.hpp (hunk 2 above)
Comment (rasapala) [reply src/python/pythonnoderesources.hpp:51]: It was windows compilation problem. I have found a fix. We need to change the resource definition from struct to class. Otherwise, static member functions with definition in cpp files are not visible.

File: src/BUILD [hunk 4]
```diff
@@ -457,12 +457,15 @@ cc_library(
             "//:disable_mediapipe" : [],
         }),
         deps = select({
-            "//:not_disable_python": [
-                "//src/python:libovmspythonmodule",
+            "//:is_windows_or_python_is_disabled_no_llm": [],
```
[the remark this answers is not in the document]
Comment (rasapala) [reply src/BUILD:460]: Checked it builds and executes the tests from this select on linux.

**Review by atobiszei — APPROVED** [review]
